docs/portal/image-registry/adding-projects.md

```diff
@@ -0,0 +1,17 @@
+# Adding Projects
+
+Admin portal users can use the following procedure to add projects.
+**Note:** Non-admin portal users are not permitted to add projects.
```

Comment:
This is a bit inconsistent with previous documentation pages. Previously, such notes are always placed at the bottom of a  page. This page "updating-image-pull-secret" is an example.

Reply:
I placed this note at the top of the topic because I felt it was important to alert non-admin users right from the start that they could not perform the procedure of adding projects. If I placed the note at the bottom of the topic, non-admin users might not even see it unless they parse through the entire topic.

As for consistency, I added a <cr> before the note. I will do the same for the other notes. Thank you!

Reply:
I placed the note at the beginning of the topic, so it would be evident to non-admin users that they cannot perform this procedure. Otherwise, they might wonder why they cannot perform the procedure, unless they happen to scroll to the bottom of the topic to see the note.

Reply:
OK, that's fine. We can keep it at the top.